components/input/index.jsx

```diff
@@ -294,7 +325,39 @@ class Input extends React.Component {
 
 	getErrorId = () => this.props['aria-describedby'] || this.generatedErrorId;
 
-	// This is convuluted to maintain backwards compatibility. Please remove deprecatedProps on next breaking change.
+	getCounterButtonIcon = (direction) => {
+		const stopStepping = () => {
+			clearTimeout(this.stepping.timeout);
+			this.stepping.currentDelay = this.stepping.initialDelay;
+		};
+
+		return (
+			<Button
+				assistiveText={{ icon: `${direction} ${COUNTER}` }}
```

Comment:
The words needs to come from a prop. `assistiveText.increment` and `assistiveText.decrement`